sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -138,25 +129,25 @@ object DateTimeUtils {
    * Converts the timestamp to milliseconds since epoch. In spark timestamp values have microseconds
    * precision, so this conversion is lossy.
    */
-  def toMillis(us: SQLTimestamp): Long = {
+  def toMillis(us: Long): Long = {
```

Comment:
nit: shall we make the name clearer? `microsToMillis`

Reply:
I renamed it in another PR already: https://github.com/apache/spark/pull/27618